test/request.spec.js

```diff
@@ -758,14 +758,13 @@ describe('AWS.Request', function() {
         s = request.createReadStream();
         s.on('error', function(e) {
           error = e;
-          expect(error).to.be["null"];
+          expect(error).to.not.be["null"];
         });
         s.on('data', function(c) {
           return data += c.toString();
         });
         return s.on('end', function() {
-          expect(error).to.be["null"];
-          expect(data).to.equal('FOOBARBAZQU');
+          expect(error).to.not.be["null"];
```

Comment:
Is this a breaking change? Previously, users would get up to `content-length` bytes and no more if the returned stream exceeded that length; now, the stream will emit an error event in that case. Was there ever a case where `content-length` would be wrong but the user was OK with receiving a truncated stream?